WordPress/Classes/ViewRelated/System/WPTabBarController+QuickStart.swift

```diff
@@ -10,17 +9,26 @@ extension WPTabBarController {
 
             guard let userInfo = notification.userInfo,
                 let element = userInfo[QuickStartTourGuide.notificationElementKey] as? QuickStartTourElement,
-                [.readerTab].contains(element),
-                let tabBar = self?.tabBar else {
+                [.readerTab].contains(element) else {
                     return
             }
 
             let newSpotlight = QuickStartSpotlightView()
```

Comment:
I wonder if we should move the code in this observer block into a separate method, just to make things a little cleaner.

Reply:
So the method `startWatchingQuickTours` already does basically just that. I can create another method to include only what's in the closure, but then it would be a method with a notification and a view as arguments, it would sort of look like the old way using a `selector` instead of the closure. Do you mind if we leave it as it is?

Reply:
Sure, totally fine!